internal/webapi/server_test.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/gpuctl/gpuctl/internal/database"
 	"github.com/gpuctl/gpuctl/internal/uplink"
 	"github.com/gpuctl/gpuctl/internal/webapi"
+	"github.com/stretchr/testify/assert"
```

Comment:
NIT: I'd like to keep 3rd party imports in a seperate group to 1st party (and std)